frontend/packages/dev-console/integration-tests/support/step-definitions/crw/crw-basic.ts

```diff
@@ -0,0 +1,44 @@
+import { Then, When } from 'cypress-cucumber-preprocessor/steps';
+import { submitButton } from '@console/cypress-integration-tests/views/form';
+import { authorizeAccessPage } from '../../pages/crw/authorizeAccess';
+import { keycloakRegistrationPage } from '../../pages/crw/keycloak';
+
+When('user click on Codeready Workspaces in Application menu in Masthead', () => {
+  cy.byTestID('application-launcher-item')
+    .should('have.attr', 'target', '_blank')
+    .invoke('removeAttr', 'target')
+    .click();
+});
+
+Then('user is redirected to Codeready Workspaces Dashboard', () => {
+  const idp = Cypress.env('BRIDGE_HTPASSWD_IDP') || 'test';
+  const username = Cypress.env('BRIDGE_HTPASSWD_USERNAME') || 'test';
+  const password = Cypress.env('BRIDGE_HTPASSWD_PASSWORD') || 'test';
+  cy.byLegacyTestID('login').should('be.visible');
+  cy.contains(idp)
+    .should('be.visible')
+    .click();
+  cy.get('#inputUsername').type(username);
+  cy.get('#inputPassword').type(password);
+  cy.get(submitButton).click();
```

Comment:
why do we need to login every time

Reply:
Codeready Workspaces uses Openshift OAuth for logging users in... So when the test opens CRW URL, it is redirected to Openshift login page, where it needs to log in (even though the test is already logged in in web console). After successfull login, user is also requested to check permissions (this is handled on lines #24 to #34)